heal-remedy.lic

```diff
@@ -91,9 +91,12 @@ def remedy_apply_wounds(body_part)
     if DRCI.get_item?(remedy, @remedy_container)
       case remedy
       when /salve/, /ungent/, /poultices/, /ointment/
-        DRC.bput("rub #{remedy}", 'You')
+        DRC.bput("rub my #{remedy}", 'You')
       when /potion/, /tonic/, /elixir/, /draught/
-        DRC.bput("drink #{remedy}", 'You')
+        case DRC.bput("drink my #{remedy}", "You drink", "Drink what?", "can't drink")
```

Comment:
Feel like this would make a good little helper method:
```
consume_remedy?(remedy_name, try_drink_first)
  if try_drink_first
     drink
        when fail, eat
  else
    eat
      when fail, drink
  end
end
```

Reply:
Agree

Reply:
Addressed per https://github.com/rpherbig/dr-scripts/pull/4966/commits/a76fe0555590fd1f8f702eded9f64ce5f9fdb2ee